e2e/test/scenarios/collections/archive.cy.spec.js

```diff
@@ -15,6 +19,42 @@ describe("scenarios > collections > archive", () => {
     cy.signInAsAdmin();
   });
 
+  it("should hide read-only archived items (metabase#24018)", () => {
+    const READ_ONLY_NAME = "read-only dashboard";
+    const CURATEABLE_NAME = "curate-able dashboard";
+
+    // setup archive with read-only collection items
+    createAndArchiveDashboard({
+      name: READ_ONLY_NAME,
+      collection_id: null,
+    });
+
+    // setup archive with curate-able collection items (user created items)
+    cy.signIn("readonly");
+
+    createAndArchiveDashboard({
+      name: CURATEABLE_NAME,
+      collection_id: 6,
```

Comment:
Is there a constant/variable with the readonly user's `collection-id`? I would prefer not to hard-code it.